infra/main.bicep

```diff
@@ -12,16 +12,23 @@ param location string
 @description('Relative Path of ASA Jar')
 param relativePath string
 
+@allowed([
+  'Consumption'
+  'Standard'
+])
+param plan string
+
 @secure()
 @description('PSQL Server administrator password')
-param psqlAdminPassword string
+param psqlAdminPassword string = 'SuperAdminPassword1!'
```

Comment:
It can't generate random password in AZD Bicep without KeyVault, and for now, we don't use KeyVault to store psql password (ASA consumption plan don't support MI). Thus we use hard code to set password.

When this template support KeyVault in the future, we should use random password replace of hard code.